### PR TITLE
Address MS Learn docs build system warning and suggestion

### PR DESCRIPTION
**Pull request by erikadoyle**
## Description

Minor fixes to address Microsoft Learn (API reference docs) build system warning and suggestion.

Fixes have been implemented (hard coded), validated, and published in generated yml doc source and published live. This PR incorporates the fix into source so it will continue to generate correctly in future doc updates.

### Main changes in the PR:

1. In **call.ts**, escape the open-angle bracket, as the markdown parser was dropping it as invalid html
2. In **tasks.ts**, update the link in the extended description from *TaskModuleDimension* to [DialogDimension](https://learn.microsoft.com/en-us/javascript/api/@microsoft/teams-js/dialogdimension?view=msteams-client-js-latest). TaskModuleDimension is no longer exported in v2 and thus has no corresponding API reference page. Note that the link text still remains "TaskModuleDimension", as that seems faithful to the [underlying source](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/packages/teams-js/src/public/constants.ts#L100).

## Validation

Fixes have been implemented (hard coded), validated, and published in the generated yml doc source and published live. This PR incorporates the fix into the source so it will continue to generate correctly in future doc updates.

### Validation performed:

1. From [getDefaultSizeIfNotProvided(TaskInfo)](https://learn.microsoft.com/en-us/javascript/api/@microsoft/teams-js/tasks?view=msteams-client-js-latest#@microsoft-teams-js-tasks-getdefaultsizeifnotprovided), click on the link with text: **TaskModuleDimension.Small**. Its no longer a broken link because it resolves to DialogDimension page.
2. From [StartCallParams `targets` property description](https://learn.microsoft.com/en-us/javascript/api/@microsoft/teams-js/call.startcallparams?view=msteams-client-js-latest#@microsoft-teams-js-call-startcallparams-targets), notices that this string and enclosing angle brackets now render in the description: `<phonenumber>`

### Unit Tests added:

No

### End-to-end tests added:

N/A

### Change file added:

Yes

### Related PRs:

https://github.com/OfficeDev/msteams-client-docs-ref-typescript/pull/123/files

### CODE REVIEW

**Review by TrevorJoelHarris — DISMISSED** [review]

File: packages/teams-js/src/public/tasks.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ export namespace tasks {
 
   /**
    * Sets the height and width of the {@link TaskInfo} object to the original height and width, if initially specified,
-   * otherwise uses the height and width values corresponding to {@link TaskModuleDimension | TaskModuleDimension.Small}
+   * otherwise uses the height and width values corresponding to {@link DialogDimension | TaskModuleDimension.Small}
```
Comment (AE-MS) [inline packages/teams-js/src/public/tasks.ts:130]: This is kind of funny and maybe it's just a docs issue. It seems like `TaskModuleDimension` is still defined and exported:
https://github.com/OfficeDev/microsoft-teams-library-js/blob/993debfdc0dfba68bdf3126bc936627c210cc5bd/packages/teams-js/src/public/constants.ts#L100
...but it's really just an alias (re-export) of the `DialogDimension` type. Do re-exports like this not show up in documentation?

**Review by AE-MS — CHANGES_REQUESTED** [review]
Waiting for changefile addition and answer to question.  😀

File: packages/teams-js/src/public/call.ts
[the remark this answers is not in the document]
Comment (erikadoyle) [reply packages/teams-js/src/public/call.ts:1]: When I ran `pnpm beachball` it returned "No change files are needed". I pulled the latest from main so hopefully that resolves the build issues.

[the remark this answers is not in the document]
Comment (AE-MS) [reply packages/teams-js/src/public/call.ts:1]: I am 99.9% sure that this requires a changefile. Usually when you get this message, it means that your fork's main branch isn't up-to-date with the upstream's main branch.

If you go to the [main branch for your fork page on github](https://github.com/erikadoyle/microsoft-teams-library-js/tree/main) and press this "update branch" button:
image
...then try to run `pnpm run changefile` does it still tell you nothing is needed?

**Review by AE-MS — DISMISSED** [review]

File: packages/teams-js/src/public/call.ts
[the remark this answers is not in the document]
Comment (AE-MS) [reply packages/teams-js/src/public/call.ts:1]: You may also have to pull main locally after doing that update but I'm not sure.

[the remark this answers is not in the document]
Comment (erikadoyle) [reply packages/teams-js/src/public/call.ts:1]: Ah! I was running the command from repo root rather than teams-js. Just pushed my changefile.

**Review by AE-MS — APPROVED** [review]

**Review by TrevorJoelHarris — APPROVED** [review]

**Pull request merged**